src/index.js

```diff
@@ -6,13 +6,16 @@ import parser from './parser'
 import doApiCall from './api';
 
 let logger = log4js.getLogger();
-logger.level = 'debug';
+logger.level = process.env.DEBUG_LEVEL || 'warn';
```

Comment:
Maybe info by default ? I don't know really, because if I do not specify any logger level I don't even get the log saying the server is up and listening on port...